src/olympia/reviewers/forms.py

```diff
@@ -114,6 +115,28 @@ class VersionsChoiceWidget(forms.SelectMultiple):
         amo.STATUS_AWAITING_REVIEW: ('reject_multiple_versions',),
     }
 
+    def get_extended_status(self, obj):
+        if not obj or not obj.file:
+            return
+        # This is the default status
+        status = obj.file.get_status_display()
+        # But we override with more a specific status if available
+        if (
+            reviewer_flags := getattr(obj, 'reviewerflags', None)
+        ) and reviewer_flags.pending_rejection:
+            status = 'Delayed Rejection'
```

Comment:
Does this mean the rejection has not been executed yet?

Reply:
yes, `pending_rejection` should be a (truthy) date when it's been set but not yet rejected.

Reply:
Maybe renaming it to the following might make it more clear:

_Delay-rejected, scheduled for <DATE\>_

Reply:
sure, we can... but as I understood it the purpose of the issue was to notate the version ids to know which should/could be rejected.  Does knowing the scheduled date add anything to that?

Reply:
https://github.com/mozilla/addons-server/pull/19465/commits/a9e9d769fac6b178db0b8ecaf6e594307d4e4ac7